internal/contribroutines/contribroutines.go

```diff
@@ -0,0 +1,34 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+package contribroutines
+
+import "sync"
+
+var (
+	stop chan struct{}
+	once sync.Once
+	mu   sync.Mutex
+)
+
+func InitStopChan() {
```

Comment:
```suggestion
func Init() {
```

Just in case this package evolves and needs to initialize more things beyond the stop chan.